actionview/app/assets/javascripts/README.md

```diff
@@ -36,6 +36,16 @@ Require `rails-ujs` into your application.js manifest.
 //= require rails-ujs
 ```
 
+Usage with yarn/webpacker
```

Comment:
Sorry I think I miss read it.  Can change it to  node_modules instead pf npm package, but do you think that may be confusing when you mention installing through Yarn previously in the readme?

I could update Installation using Yarn to include npm install instructions too then that might make it a little clearer maybe?